guides/source/security.md

```diff
@@ -1215,4 +1215,5 @@ The security landscape shifts and it is important to keep up to date, because mi
 * Subscribe to the Rails security [mailing list](https://groups.google.com/forum/#!forum/rubyonrails-security).
 * [Brakeman - Rails Security Scanner](https://brakemanscanner.org/) - To perform static security analysis for Rails applications.
 * [Keep up to date on the other application layers](http://secunia.com/) (they have a weekly newsletter, too).
+* Mozilla has [web security guidelines](https://infosec.mozilla.org/guidelines/web_security.html) that cover various HTTP headers, cookies and TLS configuration.
```

Comment:
```suggestion
* [Mozilla's Web Security Guidelines](https://infosec.mozilla.org/guidelines/web_security.html) - Recommendations on topics covering Content Security Policy, HTTP headers, Cookies, TLS configuration, etc.
```

Reply:
:+1:

Reply:
The Secunia link right above my change is actually kind of broke. I think it should probably be https://www.flexera.com/resources.html?type=blog ?

Reply:
Yeah I was going to remove it separately. You can go ahead and remove in this one, it points to some products now, not super useful.

Reply:
Done :+1: I think the Infosec documents are quite good in many ways.